### PR TITLE
Eth2Node.stop(): trace msg on timeout

### PR DESCRIPTION
**Pull request by stefantalpalaru**

### CODE REVIEW

File: beacon_chain/eth2_network.nim [hunk 1]
```diff
@@ -738,14 +738,17 @@ proc start*(node: Eth2Node) {.async.} =
...
-  # Ignore errors in futures, since we're shutting down.
-  # Use a timer to avoid hangups.
-  discard await one(sleepAsync(5.seconds),
-                    allFutures(@[
-                      node.discovery.closeWait(),
-                      node.switch.stop(),
-                    ])
-    )
+  # Ignore errors in futures, since we're shutting down (but log them on the
+  # TRACE level, if a timeout is reached).
```
Comment (arnetheduck) [inline beacon_chain/eth2_network.nim:742]: we can even do `debug` here, it shouldn't really happen after all..

Comment (stefantalpalaru) [reply beacon_chain/eth2_network.nim:742]: I don't want regular users seeing error messages on Ctrl+C. It defeats the whole concept of graceful SIGINT handling.

Comment (arnetheduck) [reply beacon_chain/eth2_network.nim:742]: the intent is that regular users run INF on console and DBG to file

Comment (stefantalpalaru) [reply beacon_chain/eth2_network.nim:742]: https://github.com/status-im/nim-beacon-chain/blob/b28a0378d1c0960e0684fcdb9b2a0cf36134281e/beacon_chain/conf.nim#L38

**Pull request merged**